### PR TITLE
Fix example settings.json to be loadable

### PR DESCRIPTION
**Pull request by rasmussen-untangle**
Updated example settings.json to be loadable by the policy manager loader.

### CODE REVIEW

File: v1/policy_manager/test_settings.json [hunk 1]
```diff
@@ -791,7 +773,7 @@
                 "name": "Default CP",
                 "description": "",
                 "enabled": true,
-                "type": "CaptivePortalRuleObject",
+                "type": "mfw-rule-captiveportal",
                 "conditions": [
                     "ef062718-ecb5-4592-9502-9b61482ea6a6"
                 ],
```
Comment (jsommerville-untangle) [inline v1/policy_manager/test_settings.json:779]: I think we should also be sure to add the 'key' property to any actions that are service level configurations (SET_CONFIGURATION types)

File: v1/policy_manager/test_settings.json [hunk 2]
```diff
@@ -481,20 +464,20 @@
...
             "name": "Policy 8/28/2023 8:02:00 PM",
             "description": "",
-            "type": "ConditionObject",
+            "type": "mfw-object-condition",
             "items": [
                 {
                 "type": "DESTINATION_ADDRESS",
                 "op": "==",
-                "value": "192.168.2.10-192.168.2.30"
+                "value": ["192.168.2.0/24"]
```
Comment (jsommerville-untangle) [inline v1/policy_manager/test_settings.json:472]: is IP range not working?

File: v1/policy_manager/test_settings.json [hunk 3]
```diff
@@ -273,41 +262,35 @@
...
             "description": "",
-            "type": "ServiceEndpointObject",
+            "type": "mfw-object-service",
             "items": [
                 {
-                "port": "3306",
-                "protocol": [
-                    "6"
-                ]
+                "port": 3306,
```
Comment (jsommerville-untangle) [inline v1/policy_manager/test_settings.json:284]: these should both be arrays also

File: v1/policy_manager/test_settings.json [hunk 4]
```diff
@@ -203,50 +203,39 @@
...
+            "type": "mfw-object-service",
             "items": [
                 {
-                "port": "88",
-                "protocol": [
-                    "6",
-                    "17"
-                ]
+                "port": 88,
+                "protocol": 17
```
Comment (jsommerville-untangle) [inline v1/policy_manager/test_settings.json:222]: should be arrays

File: v1/policy_manager/test_settings.json (hunk 3 above)
Comment (rasmussen-untangle) [reply v1/policy_manager/test_settings.json:284]: They are service end points?
type ServiceEndpoint struct {
    Protocol uint `json:"protocol"`
    Port     uint `json:"port"`
}
Loader will not load them if they are arrays. Should we change that ?

File: v1/policy_manager/test_settings.json (hunk 4 above)
Comment (rasmussen-untangle) [reply v1/policy_manager/test_settings.json:222]: They are ServiceEndPoints which is not currently defined to have arrays for those elements.

File: v1/policy_manager/test_settings.json (hunk 2 above)
Comment (rasmussen-untangle) [reply v1/policy_manager/test_settings.json:472]: No, complains its not a proper CIDR format. Did think we had fixed it at some point.

File: v1/policy_manager/test_settings.json (hunk 4 above)
Comment (rasmussen-untangle) [reply v1/policy_manager/test_settings.json]: Fix data, loading pending MFW-3619

File: v1/policy_manager/test_settings.json [hunk 5]
```diff
@@ -510,7 +504,7 @@
             "id": "4c6c51ca-02a0-4cb0-92a9-9d90a569b9f5",
             "name": "Policy 8/28/2023 8:04:15 PM",
             "description": "",
-            "type": "ConditionObject",
+            "type": "mfw-object-condition",
             "items": [
                 {
                 "type": "SERVICE",
```
[the remark this answers is not in the document]
Comment (rasmussen-untangle) [reply v1/policy_manager/test_settings.json:510]: It's not. Nice catch. Thanks.

**Pull request merged**